client/src/pages/Families/FamiliesPages.tsx

```diff
@@ -7,7 +7,7 @@ import { NewMateType } from "../types";
 
 /* Hardcoded data below */
 const family = "College";
-const syncs: NewMateType[] = [
+export const syncs: NewMateType[] = [
```

Comment:
borrowing this for mock timeline on `DashboardPage.tsx`

Reply:
lets move this to some test.ts file or fixture.ts file as well.